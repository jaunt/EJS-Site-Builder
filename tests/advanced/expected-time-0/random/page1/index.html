<html lang="en">
<meta name="description" content="wrappers/alternate">

<head>
  <title>1</title>
</head>

<body>
  Alternate:
  <i> <div>hello 1</div> </i>
</body>

</html>