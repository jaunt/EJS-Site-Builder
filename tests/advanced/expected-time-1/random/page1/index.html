<html lang="en">
<meta name="description" content="wrappers/alternate updated">

<head>
  <title>1</title>
</head>

<body>
  Alternate wrapper:
  <i> <div>hello 1</div> </i>
</body>

</html>