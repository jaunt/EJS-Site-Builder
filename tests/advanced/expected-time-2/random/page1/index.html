<html lang="en">

<head>
  <title>1</title>
</head>

<div class="but">about me</div>
<div>My buttons are clicky at times</div>

grape,apple
Fruit: grape

grape,apple
Fruit: apple


<body>
  Alternate:
  <i> <div>s hello 21</div> </i>
</body>

</html>