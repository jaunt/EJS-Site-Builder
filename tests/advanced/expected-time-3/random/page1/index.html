Double Wrap Start...

Widget Wrapper start...
<div class="but">about me</div>
<div>My wrapped buttons are clicky at stuff</div>

grape,apple
Fruit: grape

grape,apple
Fruit: apple

...end

<html lang="en">
<meta name="description" content="wrappers/alternate wrapped">

<head>
  <title>1</title>
</head>

<div class="but">about me</div>
<div>My buttons are clicky at times</div>

grape,apple
Fruit: grape

grape,apple
Fruit: apple


<body>
  Alternate:
  <i> <div> sync return nothing hello 21</div> </i>
</body>

</html>

...End