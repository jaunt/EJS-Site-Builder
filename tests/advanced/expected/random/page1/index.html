<html lang="en">

<head>
  <title>1</title>
</head>

<body>
  Alternate:
  <i> <div>1637038322852</div> </i>
</body>

</html>